Frontend/CO.CDP.OrganisationApp/Pages/Supplier/LegalFormOtherOrganisation.cshtml.cs

```diff
@@ -13,7 +14,7 @@ public class LegalFormOtherOrganisationModel(
         IOrganisationClient organisationClient) : PageModel
 {
     [BindProperty]
-    [Required(ErrorMessage = "Please select your organisation's registered legal form")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Supplier_LegalFormOtherOrganisation_PleaseSelect))]
```

Comment:
Use following pattern for Required attribute

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/main/docs/development/cookbooks/02-how-to-localize-ui-text.adoc#how-to-localize-form-validation-errors